spellcheck.pl

```diff
@@ -168,10 +182,18 @@ sub spellcheck_key_pressed
 
     return unless defined $suggestions;
 
+    my $color = Irssi::settings_get_str('spellcheck_word_input_color');
+    if ($color && Irssi->can('gui_input_set_extents')) {
+        Irssi::gui_input_set_extents($start, length $word, $color, '%n');
+    }
+
+    return unless Irssi::settings_get_bool('spellcheck_print_suggestions');
+
     # show corrections window if hidden
     if ($correction_window) {
-        $win->command("window show $window_name");
-        $correction_window->command('window stick off');
+        $win->command("^window show $window_name");
+        $correction_window->command('^window stick off');
```

Comment:
What does leading `^` do?

Reply:
hide messages like window is not sticky/window is already visible